### DISCUSSION TITLE
close  window

### DISCUSSION THREAD

**Discussion by Vjze** [marc2332/freya#611] [Q&A] [answered]
I'm building a software using freya, but I can't find an API about button clicking to close the window, can someone tell me?

**Comment by marc2332** [comment]
That sounds like a good feature to have, I'll implement it later today!

**Comment by marc2332** [comment, marked as answer]
Just added it on https://github.com/marc2332/freya/pull/613, can you give it a try ? 😄

**Reply by Vjze** [reply]
Yes, it worked, thanks for your work。
